src/main/java/seedu/opus/logic/commands/SortCommand.java

```diff
@@ -22,6 +25,11 @@ public SortCommand(String keyword) {
     @Override
     public CommandResult execute() {
         model.sortList(keyword);
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (keyword.contains("all") || keyword.contains("status") || keyword.contains("priority")
```

Comment:
IMO it's cleaner to put these commands into an `Array`, iterate through it and if `keyword` contains that command, then return.

Small matter, it's fine without changing too :)